Backend/Routes/AppRoutes/Project.route.js

```diff
@@ -1,47 +0,0 @@
-const {Router} = require('express');
-const {ProjectModel} = require('../../Models/Task.model');
-
-const ProjectController = Router();
-
-
-ProjectController.post('/', async(req,res)=>{
-    const {project_name, user_id} = req.body;
-    const project = new ProjectModel({
-        project_name,
-        user_id
-    })
-    await project.save();
-    return res.send({message : " new project added"})
-})
-ProjectController.get("/", async(req,res)=>{
-    const {user_id} = req.body;
-    const project = await ProjectModel.find();
-    
-   return res.send({project})
-});
-
-ProjectController.patch('/:projectid/edit',async(req,res)=>{
-    const {project_id, user_id, project_name} = req.body;
-    const project = await ProjectModel.find({project_id});
-    if(project.user_id===user_id){
-        const newProject = await ProjectModel.findByIdAndUpdate({_id : project_id}, req.body, {new : true});
-        return res.status(200).send({message : "project updated successfully", newProject});
-
-    }else{
-        return res.status(401).send("you are not authorised to update project")
```

Comment:
while sending an error message, keep the same format

`{message : "you are not authorized to update project"}`